routes/web.php

```diff
@@ -83,6 +83,11 @@
     Route::get('post', [PostController::class, 'index'])->name('post.index');
     Route::post('post', [PostController::class, 'store'])->name('post.store');
 
+    Route::get('post/edit/{id}', [PostController::class, 'edit'])->name('post.edit');
+    Route::post('post/update', [PostController::class, 'update'])->name('post.update');
```

Comment:
Any reason not to specify the id in the route? (like for the get and delete)
```suggestion
    Route::post('post/update/{post}', [PostController::class, 'update'])->name('post.update');
```